Sources/Nimble/Matchers/Contain.swift

```diff
@@ -61,11 +60,12 @@ private func contain(items: [AnyObject?]) -> NonNilMatcherFunc<NMBContainer> {
         failureMessage.postfixMessage = "contain <\(arrayAsString(items))>"
         let actual = try actualExpression.evaluate()
         return all(items) { item in
-            return actual != nil && actual!.containsObject(item)
+            return actual != nil && actual!.containsObject(item!)
```

Comment:
I think this expression needs to be refactored so that `!` doesn't cause an assertion failure.